lib/rules/no-anonymous-once.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @fileOverview Disallow the use of anonymous functions passed to scheduleOnce and once.
+ */
+'use strict';
+
+const { getCaller } = require('../utils/caller');
+const MESSAGE = 'The uniqueness once offers is based on function uniqueness, each invocation of this line will always create a new function instance, resulting in uniqueness checks, that will never be hit. Please replace with a named function.';
```

Comment:
I was not sure if you meant the ember docs or the guides that exist inside this repo. Since the ember docs mention this, I added a the link to the `scheduleOnce` docs.